### PR TITLE
Fix some links with http and no https

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fix some links with http and no https
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | I think QA is not needed 🤔

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17257)

### CODE REVIEW

**Review by Progi1984 — APPROVED** [review]
Need a second review before merge. No need QA.

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @matks

**Pull request merged**
